spec/requests/ndbn_members_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe "NDBNMembers", type: :request do
+  let(:user) { create(:super_admin) }
+
+  before do
+    sign_in user
+  end
+
+  describe "GET /index" do
+    it "renders the index page" do
+      get admin_ndbn_members_path
+
+      expect(response).to be_successful
+    end
+
+    it "displays the NDBN members and csv upload" do
+      create(:ndbn_member, ndbn_member_id: "123", account_name: "A Baby Center")
+
+      get admin_ndbn_members_path
+
+      html = Nokogiri::HTML(response.body)
+
+      expect(html.css("h1").text).to eq("NDBN Member Upload")
+      expect(html.css("input[type=file]").count).to eq(1)
+      expect(html.css("button[type=submit]").count).to eq(1)
+
+      expect(html.css("th").map(&:text)).to match_array(["NDBN Member Number", "NDBN Member Name"])
+      expect(html.css("tbody tr td").map(&:text)).to match_array(["123", "A Baby Center"])
+    end
+  end
+
+  describe "POST /create" do
+    it "updates the index contents" do
+      params = {member_file: fixture_file_upload("spec/fixtures/ndbn-large-import.csv", "text/csv")}
```

Comment:
Why do we need to test a 90-line CSV file? A couple of lines should be enough.

Reply:
Do you think the small CSV test is enough? I was doing it just to be thorough. I'm pretty sure the small CSV test has all the edge cases that are worth testing.

Reply:
Yep, I think the small one is fine.